src/plugin.c

```diff
@@ -253,6 +253,56 @@ static int register_callback (llist_t **list, /* {{{ */
 	return (0);
 } /* }}} int register_callback */
 
+static void log_list_callbacks (llist_t **list, /* {{{ */
+				const char *comment)
+{
+	char *str;
+	int len;
+	llentry_t *le;
+	int i;
+	int n;
+	char **keys;
+
+	n = llist_size(*list);
```

Comment:
Since I developed this from a situation where I needed this Information, I can tell you it will get logged often; Essentially collectd isn't in a workable state then.
I first had implemented it the way you sugest, and it wouldn't deliver any usefull information anymore, but simply flood the log.